wgpu-core/src/device/mod.rs

```diff
@@ -2839,6 +2887,16 @@ impl<G: GlobalIdentityHandlerFactory> Global<G> {
         device_id: id::DeviceId,
         desc: &resource::TextureDescriptor,
         id_in: Input<G, id::TextureId>,
+    ) -> (id::TextureId, Option<resource::CreateTextureError>) {
+        self.device_create_texture_raw_image::<B>(device_id, desc, None, id_in)
+    }
+
+    pub fn device_create_texture_raw_image<B: GfxBackend>(
```

Comment:
The logic here is modeled from https://github.com/Ralith/openxrs/blob/5283daa09ddbe7170069ff3ea9e42e8a8a9ff4bc/openxr/examples/vulkan.rs#L537 - basically a raw image is required for the swapchain